packages-ts/gauntlet-terra-cw-plus/src/lib/types.ts

```diff
@@ -14,19 +14,30 @@ export enum Action {
   NONE = 'none',
 }
 
+type Coin = {
+  denom: string
+  amount: string
+}
+
 export type WasmMsg = {
   wasm: {
     execute: {
       contract_addr: string
-      funds: {
-        denom: string
-        amount: string
-      }[]
+      funds: Coin[]
       msg: string
     }
   }
 }
 
+export type BankMsg = {
```

Comment:
These are duplicates of the `WasmMsg` and `BankMsg` as defined in the SDK. I'm trying to understand why do we declare types of the same name instead of using the SDK ones directly.

Reply:
Only the naming is the same, but the type is different. This are the messages to the multisig, see the [CW3 Wasm schema](https://github.com/smartcontractkit/chainlink-terra/blob/develop/packages-ts/gauntlet-terra-contracts/artifacts/contracts/cw3_flex_multisig/schema/execute_msg.json#L522) for example